packages/web-api/src/WebClient.spec.ts

```diff
@@ -801,7 +801,7 @@ describe('WebClient', () => {
 
   describe('has an option to set request concurrency', () => {
     // TODO: factor out common logic into test helpers
-    const responseDelay = 100; // ms
+    const responseDelay = 500; // ms
```

Comment:
On windows I think the delay is too short and the tests assertions are failing, bumping this value seems to make the test stable

Reply:
This makes sense to me but I'm open to revisitng this in #2221 👀